rust/src/core.rs

```diff
@@ -8,6 +8,12 @@ use crate::ffiutil;
 use ffiutil::*;
 use openat_ext::OpenatDirExt;
 
+/// The binary forked from useradd that pokes the sss cache.
+/// It spews warnings (and sometimes fatal errors) when used
+/// in a non-systemd container (default treecompose side) so
+/// we replace it with `true`.
```

Comment:
This says we replace with `true`, but AFAICT the code below only renames. Is the code missing that bit, or should that comment be updated?